core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * @fileoverview Component for the feedback Updates page.
+ */
+
+import { Component, OnInit, ChangeDetectorRef } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { WindowRef } from 'services/contextual/window-ref.service';
+import './contributor-admin-dashboard-page.component.css';
+import { animate, state, style, transition, trigger } from '@angular/animations';
+import { ContributorDashboardAdminStatsBackendApiService } from './services/contributor-dashboard-admin-stats-backend-api.service';
+
+@Component({
+  selector: 'contributor-admin-dashboard-page',
+  styleUrls: ['./contributor-admin-dashboard-page.component.css'],
+  templateUrl: './contributor-admin-dashboard-page.component.html',
+  animations: [
+    trigger('detailExpand', [
+      state('collapsed', style({height: '0px', minHeight: '0'})),
+      state('expanded', style({height: '*'})),
+      transition(
+        'expanded <=> collapsed',
+        animate('225ms cubic-bezier(0.4, 0.0, 0.2, 1)')),
+    ]),
+  ],
+})
+export class ContributorAdminDashboardPageComponent implements OnInit {
+  activeTab!: string;
+  TAB_NAME_TRANSLATION_SUBMITTER: string = 'Translation Submitter';
+  TAB_NAME_TRANSLATION_REVIEWER: string = 'Translation Reviewer';
+  TAB_NAME_QUESTION_SUBMITTER: string = 'Question Submitter';
+  TAB_NAME_QUESTION_REVIEWER: string = 'Question Reviewer';
+  translationReviewersCount: number = 24;
```

Comment:
Why are these 24? Here and below. Can they be const somewhere?

Reply:
Sorry, it was dummy values while testing. Fixed.